base/applications/drwtsn32/main.cpp

```diff
@@ -227,6 +228,15 @@ int __stdcall wWinMain(HINSTANCE, HINSTANCE, LPWSTR cmdLine, INT)
        }
     }
 
+    if (!PathIsDirectoryW(Buffer)) {
+        int res = SHCreateDirectoryExW(NULL, Buffer, NULL);
```

Comment:
Or just call kernel32's CreateDirectory function?

Reply:
That does not create directories recursive